web/models/user.ex

```diff
@@ -35,4 +35,14 @@ defmodule RemoteRetro.User do
     |> unique_constraint(:email)
     |> validate_format(:email, ~r/@/)
   end
+
```

Comment:
@vanderhoop it's a part of this pr because we didn't want to duplicate that code in a test we were working on.  We ended up not needing the test.